config/locales/diaspora/en.yml

```diff
@@ -487,7 +489,7 @@ en:
       only_sharing_q: "Who are the people listed under “Only sharing with me” on my contacts page?"
       only_sharing_a: "These are people that have added you to one of their aspects, but who are not (yet) in any of your aspects. In other words, they are sharing with you, but you are not sharing with them: you can think of this as them “following” you. If you add them to an aspect, they will then appear under that aspect and not under “Only sharing with me”. See above."
       list_not_sharing_q: "Is there a list of people whom I have added to one of my aspects, but who have not added me to one of theirs?"
-      list_not_sharing_a: "No, but you can see whether or not someone is sharing with you by visiting their profile page. If they are, the button showing the aspect(s) in which you have placed them will be green; if not, it’ll be gray."
+      list_not_sharing_a: "No, but you can see whether or not someone is sharing with you by visiting their profile page. If they are, there will be a green check mark next to their diaspora* ID. If not, it’ll be a gray circle."
```

Comment:
No, but you can see whether or not someone is sharing with you by visiting their profile page. If they are, there will be a green **tick** next to their diaspora* ID. If **they are** not, **there will be** a gray circle.